draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -153,30 +159,8 @@ export default class Toolbar extends React.Component {
           + (selectionRect.width / 2);
       }
 
-      // ok, why we have to use external style elements here? Because, we are
-      // moving pointer position each time and it can be set via `before`,
-      // `after` css-pseudo classes
-      const pointerClassName = 'draft-js-inline-toolbar-pointer';
-      const styleId = 'draft-js-inline-toolbar-pointer-pseudo';
-
-      if (this.toolbar.classList.contains(pointerClassName)) {
-        this.toolbar.classList.remove(pointerClassName);
-      }
-      // We are recreating style element on every toolbar appearance. It is
-      // better for us to use individual style element rather than join new
-      // styles to some of the existent elements, because we can easily remove
-      // it next time
-      if (document.getElementById(`${styleId}-${this.state.nonce}`)) {
-        document.head.removeChild(
-          document.getElementById(`${styleId}-${this.state.nonce}`)
-        );
-      }
-
       if (typeof metrics.shift === 'string') {
-        const styleEl = document.createElement('style');
-        styleEl.setAttribute('id', `${styleId}-${this.state.nonce}`);
-        document.head.appendChild(styleEl);
-        this.toolbar.classList.add(pointerClassName);
+        this.toolbar.classList.add(this.state.pointerClassName);
```

Comment:
I think this is the last dom manipulation in here. Rather than doing this you could simply add `this.state.pointerClassName` to the `className` prop in the render method

Reply:
that's on line 227 - instead of `className={theme.toolbarStyles.toolbar}` you could do `className={[theme.toolbarStyles.toolbar, this.state.pointerClassName].join(' ')}

Reply:
Thanks for the tip. Done)